core/src/main/java/jenkins/agents/CloudSet.java

```diff
@@ -231,7 +231,6 @@ private void handleNewCloudPage(Descriptor<Cloud> descriptor, String name, Stapl
         checkName(name);
         JSONObject formData = req.getSubmittedForm();
         formData.put("name", name);
-        formData.put("cloudName", name); // ec2 uses that field name
```

Comment:
ec2 no longer uses that filed name
https://github.com/jenkinsci/ec2-plugin/pull/892